src/librustdoc/clean/types.rs

```diff
@@ -82,13 +82,19 @@ crate struct ExternalCrate {
 /// directly to the AST's concept of an item; it's a strict superset.
 #[derive(Clone)]
 crate struct Item {
-    /// Stringified span
-    crate source: Span,
-    /// Not everything has a name. E.g., impls
+    /// The [`Span`] of this item in the source code.
+    crate span: Span,
```

Comment:
`Span` is already linked in the autogenerated documentation, is this comment necessary? I'd rather remove it so it can't get out of date.